### PR TITLE
chore(datasource/sbt): list package versions in debug

### PR DESCRIPTION
**Pull request by cjtim**
## Changes
* use logger debug to list package versions

## Context
* I want to be able to see list of package version from remote registry

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [x] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/datasource/sbt-package/index.ts [hunk 1]
```diff
@@ -190,6 +190,7 @@ export class SbtPackageDatasource extends MavenDatasource {
 
       const dependencyUrl = searchRoot;
 
+      logger.debug({ dependency: packageName, versions }, `Package versions`);
```
Comment (viceice) [inline lib/modules/datasource/sbt-package/index.ts:193]:
```suggestion
      logger.trace({ dependency: packageName, versions }, `Package versions`);
```

File: lib/modules/datasource/sbt-plugin/index.ts [hunk 2]
```diff
@@ -119,6 +119,7 @@ export class SbtPluginDatasource extends SbtPackageDatasource {
 
       const dependencyUrl = `${searchRoot}/${artifact}`;
 
+      logger.debug({ dependency: packageName, versions }, `Package versions`);
```
Comment (viceice) [inline lib/modules/datasource/sbt-plugin/index.ts:122]:
```suggestion
      logger.trace({ dependency: packageName, versions }, `Package versions`);
```

File: lib/modules/datasource/sbt-package/index.ts (hunk 1 above)
Comment (cjtim) [reply lib/modules/datasource/sbt-package/index.ts]: done

File: lib/modules/datasource/sbt-plugin/index.ts (hunk 2 above)
Comment (cjtim) [reply lib/modules/datasource/sbt-plugin/index.ts]: done

**Comment by viceice** [comment]
please read https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting

**Review by rarkins — APPROVED** [review]

**Pull request merged**